examples/sgp4Prop/fsw/src/SGP4.c

```diff
@@ -1443,7 +1444,7 @@
         satrec->init = 'n';
 
         //sgp4fix return boolean. satrec->error contains any error codes
-        return TRUE;
+        return true;
```

Comment:
The goal of this example is to demonstrate that ECI can integrate external code without modification. Can we revert all changes to the external code itself?

Reply:
Unfortunately the external code needs to compile and since were running against a newer/latest gcc it wont compile without these changes.